pkg/algorithms/algorithm.py

```diff
@@ -60,14 +61,32 @@ def output_json(self) -> Tuple[str, str, bool]:
                     percentage_change * self.metrics_config[key]["direction"] > 0
                     or self.metrics_config[key]["direction"] == 0
                 ):
-                    dataframe_json[index]["metrics"][key][
-                        "percentage_change"
-                    ] = percentage_change
+                    dataframe_json[index]["metrics"][key]["percentage_change"] = (
+                        percentage_change
+                    )
                     dataframe_json[index]["is_changepoint"] = True
+                    if self.options["collapse"]:
+                        if (
+                            index > 0
+                            and dataframe_json[index - 1] not in collapsed_json
```

Comment:
Could we make the delta configurable, maybe default to 1, but the user could do `--collapse 3` to get the 3 data points from left and right?
If you want I can take a look at doing it in a new PR.

Reply:
Nice idea. But not a immediate priority. Lets prioritize on threshold, telco connector and making orion configuration configurable at the moment. We can certainly look at this later.